sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/_configuration.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, endpoint: str, credential: "TokenCredential", **kwargs: Any)
         self.credential = credential
         self.api_version = api_version
         self.credential_scopes = kwargs.pop("credential_scopes", ["https://cognitiveservices.azure.com/.default"])
-        kwargs.setdefault("sdk_moniker", "ai-language-conversations/{}".format(VERSION))
+        kwargs.setdefault("sdk_moniker", f"ai-language-conversations/{VERSION}")
```

Comment:
It's hard to tell, but the code in this file is actually generated via a tool called [autorest](https://www.npmjs.com/package/@autorest/python). We do not edit any files that are generated because the changes will be overwritten next time we have to regenerate the code.